sonar-batch/src/main/java/org/sonar/batch/mediumtest/FakePluginInstaller.java

```diff
@@ -22,6 +22,7 @@
 import java.io.File;
 import java.util.HashMap;
 import java.util.Map;
+import org.sonar.api.Plugin;
 import org.sonar.api.SonarPlugin;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonar.api.SonarPlugin'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUselessImportCheck)